src/sentry/static/sentry/app/views/alerts/details/header.tsx

```diff
@@ -28,7 +29,6 @@ import Status from '../status';
 type Props = {
   className?: string;
   hasIncidentDetailsError: boolean;
-  // Can be undefined when loading
```

Comment:
Is this no longer true?

Reply:
It's true, but I felt like it was pretty clear why it would be undefined. I can put it back thought.